docs/source/user/code_console.md

```diff
@@ -1,27 +1,33 @@
 
 # Code Consoles
 
-Notebooks allow you to write and run code while you create a document. At times,
-it is helpful to write and run code in an exploratory manner, while not creating
-a document; a sort of transient playground for interactive code. The JupyterLab
-Code Console addresses this usage case, with full support for the rich output of
-Jupyter.
+Code consoles allow you to run code interactively in a kernel. The cells of a
+code console show the order in which code was executed in the kernel, as opposed
+to the explicit ordering of cells in a notebook document. Code consoles also
+display rich output, just like notebook cells.
 
-Create a code console by clicking the + button in the File Browser and selecting
-the kernel:
+Create a new code console by clicking the `+` button in the file browser and
+selecting the kernel:
```

Comment:
It is also possible to create one via the `File` menu now.

Reply:
There are a lot of places in these user-facing docs that have an opinionated way to do things, where there are menu entries or other ways that accomplish the same thing. Can we address these in a separate PR?

Reply:
Sure.